visualization/dashboard/src/MLIDashboard/JointDataset.ts

```diff
@@ -121,6 +121,24 @@ export class JointDataset {
         return result;
     }
 
+    // set the appropriate error value in the keyed column
+    public static setErrorMetrics(row: {[key: string]: any}, modelType: ModelTypes): void {
```

Comment:
what about the multiclass case?

Reply:
That is not this case, this PR is for regression.